WORKSPACE

```diff
@@ -26,6 +26,20 @@ http_archive(
     ],
 )
 
+rules_kotlin_version = "v1.5.0-beta-3"
+rules_kotlin_sha = "58edd86f0f3c5b959c54e656b8e7eb0b0becabd412465c37a2078693c2571f7f"
+http_archive(
```

Comment:
Can some of this live in protobuf_deps() ? This is primarily to make usage of these rules less difficult in consuming bazel workspaces.

Reply:
Yep, moved the http_archive rule into protobuf_deps